core/src/lib.rs

```diff
@@ -58,16 +58,11 @@ cfg_client! {
 /// Shared tracing helpers to trace RPC calls.
 pub mod tracing;
 pub use async_trait::async_trait;
-pub use error::{Error, SubscriptionAcceptRejectError, SubscriptionCallbackError};
+pub use error::Error;
 
 /// JSON-RPC result.
 pub type RpcResult<T> = std::result::Result<T, Error>;
 
-/// The return type of the subscription's method for the rpc server implementation.
-///
-/// **Note**: The error does not contain any data and is discarded on drop.
-pub type SubscriptionResult = Result<(), SubscriptionCallbackError>;
```

Comment:
So, I removed this because folks have to convert their subscriptions it may be possible that they sent out subscription close errors if they doesn't read the docs carefully enough and just keep `Result<T, E>` in the subscriptions.

we could keep it as well.